core/services/pipeline/orm.go

```diff
@@ -410,6 +413,17 @@ WHERE pipeline_runs.id = batched_pipeline_runs.id`,
 		return errors.Wrap(err, "DeleteRunsOlderThan failed")
 	}
 
+	deleteTS := time.Now()
+
+	o.lggr.Debugw("pipeline_runs reaper DELETE query completed", "duration", deleteTS.Sub(start))
+	defer o.lggr.Debugw("pipeline_runs reaper VACUUM ANALYZE query completed", "duration", time.Since(deleteTS))
```

Comment:
I think that a closure is necessary to ensure that `time.Since` is not called until the defer is executed b/c:
> A deferred function’s arguments are evaluated when the defer statement is evaluated.

https://go.dev/blog/defer-panic-and-recover
```suggestion
	defer func() {
		o.lggr.Debugw("pipeline_runs reaper VACUUM ANALYZE query completed", "duration", time.Since(deleteTS))
	}()
```